merino-cache/src/memory.rs

```diff
@@ -70,7 +120,7 @@ impl<S> Suggester<S> {
     fn remove_expired_entries<K: Eq + Hash + Debug>(
         items: &Arc<DedupedMap<K, Instant, Vec<Suggestion>>>,
     ) {
-        let start = Instant::now();
+        let mut start = Instant::now();
```

Comment:
Does this need to be mutable?

Reply:
Probably not, for the same fix that you note later.